script/DOM-utils.js

```diff
@@ -0,0 +1,110 @@
+//making onewindow active at a time
+export const makeModalWindowActive = (modal) => {
+  const allOpenModals = document.querySelectorAll(".modal");
+  allOpenModals.forEach((mod) => {
+    mod.style.zIndex = "0";
+  });
+  modal.style.zIndex = "2";
+};
+//a generic function to create an element with a text, class and id if respective arguments are provided
+export const createElementWithText = (
+  tag,
+  txt = null,
+  cl = null,
+  id = null
+) => {
```

Comment:
let's test if it will work without assigining defaults, if we don;t passa value it would be undefined so falsy/truthy checks should still work